Aseprite.js

```diff
@@ -224,6 +227,34 @@ class Aseprite {
     this.colorDepth === 8 ? palette.index = this.paletteIndex : '';
     return palette;
   }
+  readSliceChunk() {
+    const numSliceKeys = this.readNextDWord();
+    const flags = this.readNextDWord();
+    const patch = (flags & 1) !== 0 ? {} : null
+    const pivot = (flags & 2) !== 0 ? {} : null
```

Comment:
If the patch and pivot data aren't present, it just shouldn't be a part of the `slice` property.